manager/controllers/app/config/evaluator_input.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package config
+
+import (
+	api "fybrik.io/fybrik/manager/apis/app/v1alpha1"
+
+	dataset "fybrik.io/fybrik/manager/controllers/app/dataset"
+	"fybrik.io/fybrik/pkg/multicluster"
+	model "fybrik.io/fybrik/pkg/taxonomy/model/base"
+)
+
+// WorkloadInfo holds workload details such as cluster/region, type, etc.
+type WorkloadInfo struct {
+	// Cluster where the user workload is running
+	Cluster multicluster.Cluster
+}
+
+// Request is a request to use a specific asset
+type Request struct {
+	// asset identifier
+	DatasetID string
+	// requested interface
+	Interface api.InterfaceDetails
+	// requested usage, e.g. "read": true, "write": false
+	Usage map[api.DataFlow]bool
+}
+
+// EvaluatorInput is an input to Configuration Policies Evaluator.
+// Used to evaluate configuration policies.
+type EvaluatorInput struct {
+	// A list of available clusters
+	Clusters []multicluster.Cluster
```

Comment:
@shlomitk1 In the future the list of available clusters should be provided as part of the infrastructure metadata, rather than being passed in as part of the EvaluatorInput.  Right? Perhaps until we have something final this information should just be in some configmap?

Reply:
@shlomitk1 I think clusters will be part of the metadata describing the infrastructure available and its costs, capabilities, etc.  I don't think it should be in the parameters passed to the evaluator, but rather stored with the rest of the infra metadata.  Where/how we store that is currently TBD (OPA data, part of the taxonomy, configmap, etc).  I would suggest adding an interface for obtaining infra metadata.  Initital implementation of that could be to obtain the clusters using ClusterManager.